plugins/woo-ai/tests/js/jest-setup.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom';
+
+beforeEach( () => {
+	// Mock fetch
+	global.fetch = jest.fn( () =>
```

Comment:
Is this required for tests to run? I think it's best to do the mocking in each test based on the use case and not globally.

Reply:
Not required; I stole this from the WooPay repository. I think the idea is that you have a global backup to prevent network requests in your unit tests, and you can always override them within your individual tests. 🤔